File: 00_ETC_PY/28701.py
n = int(input())

total = 0
for i in range(n + 1):
    total += i
print(total)

powResult = 0
powResult = total ** 2
print(powResult)

total2 = 0
for i in range(n + 1):
    threeResult = i ** 3
    total2 += threeResult
print(total2)

File: 00_ETC/28701.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main() {
    cin.tie(0)->sync_with_stdio(0);

    int n;
    cin >> n;

    int total = 0;
    for (int i = 1; i <= n; ++i) {
        total += i;
    }
    cout << total << "\n";
    int powResult = pow(total, 2);
    cout << powResult << "\n";

    int total2 = 0;
    for (int i = 1; i <= n; ++i) {
        int threeResult = pow(i, 3);
        total2 += threeResult;
    }
    cout << total2 << "\n";
    return 0;
}
//
// Created by 최승우 on 25. 11. 4.
//
